examples/example_check.py

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Would you mind adding a small docstring explaining what the purpose of this file is? It makes it a lot easier to navigate the helper files!